src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Applications.cshtml

```diff
@@ -147,8 +158,104 @@
+                                        </th>
+                                        <th class="govuk-table__header" scope="col">
+                                            <sfa-roatp-sortable-column selected-tab="appeals" column-name="AppealdetermineddDate" column-label="Application determined date" default="true" default-order="Descending" />
+                                        </th>
+                                        <th class="govuk-table__header" scope="col">
+                                            <sfa-roatp-sortable-column selected-tab="appeals" column-name="AppealSubmittedDate" column-label="Appeal submitted date" default="true" default-order="Descending" />
+                                        </th>
+                                        <th scope="col" class="govuk-table__header">Overall outcome</th>
+                                    </tr>
+                                </thead>
+                                <tbody class="govuk-table__body">
+                                    @foreach (var outcome in Model.AppealsDetails.Reviews)
+                                    {
+
+                                        <tr class="govuk-table__row">
+                                            <td data-label="Organisation name" class="govuk-table__cell">
+                                                <a class="govuk-link govuk-!-margin-bottom-0" asp-action="Appeals" asp-route-id="@outcome.ApplicationId">
+                                                    @outcome.OrganisationName
+                                                </a>
+                                            </td>
+                                            <td data-label="UKPRN" class="govuk-table__cell">@outcome.Ukprn</td>
+                                            <td data-label="Provider route" class="govuk-table__cell">@outcome.ProviderRoute</td>
+                                            <td data-label="Application reference number" class="govuk-table__cell">@outcome.ApplicationReferenceNumber</td>
+                                            <td data-label="Application submitted date" class="govuk-table__cell">@outcome.ApplicationSubmittedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Application determined date" class="govuk-table__cell">@outcome.ApplicationDeterminedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Appeal submitted date" class="govuk-table__cell">@outcome.AppealSubmittedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Overall outcome" class="govuk-table__cell">
+                                                <span class="das-no-wrap @outcome.OversightStatus.GetCssClass()">@outcome.OversightStatus.GetLabel()</span>
```

Comment:
It'll be the appeal status, not oversight status

Reply:
its part of data not the header information, ticket is for header should display respective columns, this will be corrected as part of the next ticket when we are displaying the data